head_pan_controller:
    controller:
        package: ax12_driver_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: head_pan_joint
    joint_speed: 1.17
    motor:
        id: 6
        init: 666
        min: 359
        max: 973

head_tilt_controller:
    controller:
        package: ax12_driver_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: head_tilt_joint
    joint_speed: 1.17
    motor:
        id: 5
        init: 514
        min: 275
        max: 634

